src/viur/core/modules/user.py

```diff
@@ -1409,7 +1409,7 @@ def authenticateUser(self, key: db.Key, **kwargs):
             raise ValueError(f"Unable to authenticate unknown user {key}")
 
         # Verify that this user account is active
-        if not self.is_active(skel):
+        if self.is_active(skel) is False:
```

Comment:
In this case I would keep the old behaviour. It's necessary to include the status in the viewSkel, otherwise you can't determine if it's allowed and it should fail.